src/PrestaShopBundle/Resources/views/Admin/Common/table.html.twig

```diff
@@ -0,0 +1,105 @@
+{% set filtersForm = tableView.formView %}
+
+{{ form_start(filtersForm) }}
+<table class="table" id="{{ tableView.identifier }}">
```

Comment:
ping @eternoendless, this is something I'd like to see in every listing page.

1 integration => 40+ pages migrated the same way.

Later, we may do a Vuejs integration of the system, return JSON representation of table view object.